bindings/wysiwyg-ffi/src/ffi_composer_model.rs

```diff
@@ -284,7 +284,7 @@ impl ComposerModel {
             self.inner
                 .lock()
                 .unwrap()
-                .set_link_suggestion(url, text, suggestion, attrs),
+                .set_mention_from_suggestion(url, text, suggestion, attrs),
```

Comment:
The calling function is still called `set_link_suggestion` - is that right?

Reply:
It is at the moment yes as I wasn't 100% on `set_mention_from_suggestion`. I feel like what we're actually doing is `replace_suggestion_with_mention` or `insert_mention_at_suggestion` or something like that.

Reply:
It is at the moment yes as I wasn't 100% on the name `set_mention_from_suggestion`. I feel like what we're actually doing is `replace_suggestion_with_mention` or `insert_mention_at_suggestion` or something like that.